megamek/src/megamek/common/Aero.java

```diff
@@ -2781,7 +2781,9 @@ public void autoSetThresh() {
     }
 
     public void setThresh(int val, int loc) {
-        damThresh[loc] = val;
+        if (loc < damThresh.hashCode()) {
```

Comment:
Should that be `length`?

Reply:
My brain had apparently shut off at that point.